options/ts.ts

```diff
@@ -15,6 +15,7 @@ function getTsTaskOptions(grunt: IGrunt, tsconfig: any): any {
 	const distCompilerOptionsOverride = tsOverrides && tsOverrides.dist && tsOverrides.dist.compilerOptions ? tsOverrides.dist.compilerOptions : {};
 	tsconfigDist.compilerOptions = Object.assign({}, tsconfig.compilerOptions, {
 		outDir: distDir,
+		mapRoot: '../dist/umd/_debug',
```

Comment:
previously the `sourceMaps` attributes was passed exactly as has been reverted here https://github.com/dojo/grunt-dojo2/pull/11/files#diff-bd7b43d3e04f1d87f0b3d4dbf22a69cdR78